src/codewhisperer/service/transformationResultsViewProvider.ts

```diff
@@ -137,19 +112,74 @@ export class DiffModel {
     changes: ProposedChangeNode[] = []
 
     /**
-     *
+     * This function creates a copy of the changed files of the user's project so that the diff.patch can be applied to them
+     * @param pathToWorkspace Path to the project that was transformed
+     * @param changedFiles List of files that were changed
+     * @returns Path to the folder containing the copied files
+     */
+    public copyProject(pathToWorkspace: string, changedFiles: ParsedDiff[]) {
+        const pathToTmpSrcDir = path.join(os.tmpdir(), `project-copy-${Date.now()}`)
+        fs.mkdirSync(pathToTmpSrcDir)
```

Comment:
We can punt it in this PR, but we are strictly requiring code to avoid use 'fs-extra' (example: https://github.com/aws/aws-toolkit-vscode/pull/4294 ) in order to be web-compatible.